sympy/parsing/c/c_parser.py

```diff
@@ -784,6 +845,8 @@ def transform_binary_operator(self, node):
                         # '(' is found
                         while (len(operators_stack) != 0
                             and operators_stack[-1] != '('):
+                            if len(combined_variables_stack) < 2:
```

Comment:
Can you add a reference to this in the docstring that unary operators that are children nodes of binary operators cannot be processed yet?

Reply:
Sure :)